spec/type/s3_bucket_spec.rb

```diff
@@ -49,39 +49,10 @@
   end
 
   it { should have_logging_enabled(target_bucket: 'my-log-bucket', target_prefix: 'logs/') }
-
   it { should have_tag('env').value('dev') }
-
   it { should have_versioning_enabled }
-
   it { should have_mfa_delete_enabled }
-
   it { should have_server_side_encryption(algorithm: 'aws:kms') }
-
-  context 'nested attribute call' do
-    its(:resource) { should be_an_instance_of(Awspec::ResourceReader) }
-    its('resource.name') { should eq 'my-bucket' }
```

Comment:
Why remove 'nested attribute call' test ?  and  remove https://github.com/k1LoW/awspec/pull/457/files#diff-cb79a32268707dd301cbcf04b72244afL3 ?

( see https://github.com/k1LoW/awspec/blob/master/doc/resource_types.md#unlock-advanced-use-11 )

Reply:
It effective breaks the `check_existence`... not sure why, but if memory serves me well, by only doing `aws_resource Aws::S3::Bucket` it will redirect the method call `check_existence` to `Aws::S3::Bucket` instead.
I wasn't aware of the "nested attribute call" even being documented... I assumed that the spec was outdated, since I found a repeated group over there and without loading `Aws::S3::Bucket` the examples will be broken as well.
I guess I'll have to go back and figure that out... my apologies.

Reply:
@k1LoW , I think I already fixed this item... could you please review it again?